packages/common/base/src/apiClient/CrossmintApiClient.ts

```diff
@@ -1,36 +1,45 @@
-import { validateAPIKey } from "@/apiKey/validateAPIKey";
-import { ValidateAPIKeyPrefixExpectations, ValidateAPIKeyPrefixSuccessResult } from "@/apiKey/validateAPIKeyPrefix";
+import { Crossmint } from "@/Crossmint/Crossmint";
+import { environmentToCrossmintBaseURL } from "@/apiKey/utils/environmentToCrossmintBaseURL";
+import { ValidateApiKeySuccessResult, validateAPIKey } from "@/apiKey/validateAPIKey";
+import { ValidateAPIKeyPrefixExpectations } from "@/apiKey/validateAPIKeyPrefix";
 
-import { environmentToCrossmintBaseURL } from "../apiKey/utils/environmentToCrossmintBaseURL";
 import { ApiClient } from "./ApiClient";
 
-export type CrossmintApiClientCtorParams = CrossmintApiClientCtorWithoutExpectationsParams & {
-    expectations?: ValidateAPIKeyPrefixExpectations;
-};
-
-export type CrossmintApiClientCtorWithoutExpectationsParams = {
-    apiKey: string;
-    overrideBaseUrl?: string;
-    extraAuthHeaders?: HeadersInit;
+export type CrossmintApiClientInternalConfig = {
+    sdkMetadata: {
+        name: string;
+        version: string;
```

Comment:
Is there any way this can be retrieved auotmatically from the package somehow, to ensure it's always in sync?

Reply:
inside of the individual sdks, we basically can, but not here